{
  "id": "uno-ooo-parser",
  "version": "0.1.8",
  "timestamp": "2021-12-28 10:14:21+00:00",
  "name": "ConnectionSetupException",
  "type": "exception",
  "namespace": "com.sun.star.connection",
  "parser_args": {
    "sort": true
  },
  "writer_args": {
    "include_desc": true
  },
  "data": {
    "from_imports": [
      [
        "..uno.exception",
        "Exception"
      ]
    ],
    "from_imports_typing": [],
    "quote": [],
    "typings": [],
    "requires_typing": false,
    "name": "ConnectionSetupException",
    "namespace": "com.sun.star.connection",
    "imports": [],
    "extends": [
      "com.sun.star.uno.Exception"
    ],
    "desc": [
      "Is thrown, when it is not possible to accept on a local resource."
    ],
    "url": "https://api.libreoffice.org/docs/idl/ref/exceptioncom_1_1sun_1_1star_1_1connection_1_1ConnectionSetupException.html",
    "items": {}
  }
}